imageregistry/v1/types.go

```diff
@@ -75,18 +75,21 @@ type ImageRegistrySpec struct {
 	// resources defines the resource requests+limits for the registry pod.
 	// +optional
 	Resources *corev1.ResourceRequirements `json:"resources,omitempty" protobuf:"bytes,12,opt,name=resources"`
+	// affinity is a group of node affinity scheduling rules for the image registry pod(s).
+	// +optional
+	Affinity *corev1.Affinity `json:"affinity,omitempty" protobuf:"bytes,13,opt,name=affinity"`
 	// nodeSelector defines the node selection constraints for the registry
 	// pod.
 	// +optional
-	NodeSelector map[string]string `json:"nodeSelector,omitempty" protobuf:"bytes,13,rep,name=nodeSelector"`
+	NodeSelector map[string]string `json:"nodeSelector,omitempty" protobuf:"bytes,14,rep,name=nodeSelector"`
```

Comment:
i'm pretty sure changing these is a no-no, you need to add it to the end.

Reply:
https://developers.google.com/protocol-buffers/docs/proto3#updating

Reply:
Done.